src/main/java/BadPracticesModel.java

```diff
@@ -0,0 +1,68 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class BadPracticesModel {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    // Unused variable
+    private String unusedVariable = "I am unused";
```

Comment:
The variable 'unusedVariable' is declared but never used, which can lead to confusion and clutter in the code. (Severity: low)
**Suggestion**: Remove 'unusedVariable' if it is not needed.